packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -416,6 +478,15 @@ export const handleScheduledDelivery = async (
                 schedulerId: schedulerUuid,
             },
         });
+        schedulerService.logSchedulerJob({
+            task: 'handleScheduledDelivery',
```

Comment:
I think we are missing the start and completed logs for the `handleScheduledDelivery`.
On complete, could we track the children's job ids ? so we can then relate the logs.

Reply:
I mentioned this on the description: 

> I only store handlejob logs on errors (not started or completed) , otherwise it will be confusing .

I think is going to be confusing to see this event, unless there has been an error downloading the csv/image...